smart_open/tests/test_s3.py

```diff
@@ -460,6 +460,30 @@ def test_list_bucket_long(self):
         expected = ['key_%d' % x for x in range(num_keys)]
         self.assertEqual(sorted(keys), sorted(expected))
 
+    @mock.patch('boto3.client')
+    @mock.patch('boto3.session.Session')
+    def test_credentials(self, mock_session, mock_client):
+        # setup mocks
+        mock_s3 = mock.Mock()
+        mock_client.return_value = mock_s3
+        mock_s3.list_objects_v2.return_value = {'Contents': [{'Key': 'test_key'}]}
+
+        # make call
+        # Note we disable multiprocessing since mocks only work in the thread.
+        list(smart_open.s3.iter_bucket(BUCKET_NAME,
+                                       workers=None,
+                                       aws_access_key_id='access_id',
+                                       aws_secret_access_key='access_secret'))
+
+        # asserts
```

Comment:
Please remove this. It's obvious what you're doing from the source code, so the comment adds zero value.